sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/ReadmeSamples.java

```diff
@@ -271,7 +271,12 @@ public void createIndex() {
     public void createIndexUseFieldBuilder() {
         // BEGIN: readme-sample-createIndexUseFieldBuilder
         List<SearchField> searchFields = SearchIndexClient.buildSearchFields(Hotel.class, null);
-        SEARCH_INDEX_CLIENT.createIndex(new SearchIndex("index", searchFields));
+        List<SearchField> searchFieldList = new ArrayList<>();
+        searchFieldList.add(new SearchField("hotelId", SearchFieldDataType.STRING)
```

Comment:
This can be simplified to just add the additional SearchField to `searchFields`. Another option would be adding `SimpleField` or `SearchableField` annotations to the Hotel class.

```java
@SimpleField(isKey = true, isFilterable = true, isSortable = true)
private String id;

@SearchableField(isFilterable = true, isSortable = true)
private String name;
```